app/views/staff/session_formats/update.js.erb

```diff
@@ -1,4 +1,10 @@
-$('#config-modal').modal('hide');
+<% if @session_format.save %>
```

Comment:
I noticed the call to #save below for track modal update and left a comment there. Same applies here.